### PR TITLE
Adding authentication for mobile app.

### PR DESCRIPTION
**Pull request by KiranJosh**
Added mobile app authentication.

### CODE REVIEW

File: app/controllers/api/v1/attendances_controller.rb [hunk 1]
```diff
@@ -3,6 +3,8 @@
 module Api
   module V1
     class AttendancesController < V1::BaseController
+      include AuthenticationConcern
```
Comment (pegasusflyhigh) [inline app/controllers/api/v1/attendances_controller.rb]: Skip sms_callback API from authentication.

**Comment by anilmaurya** [comment]
@KiranJosh Please app specs to validate auth functionality.

**Comment by anilmaurya** [comment]
@KiranJosh Do we need to send mobile number in params ? We are already accepting it in headers, right ?

**Comment by KiranJosh** [comment]
> @KiranJosh Do we need to send mobile number in params ? We are already accepting it in headers, right ?

Yes, Now we are not accepting the mobile number in params.

**Pull request merged**